lametro/templates/partials/meeting_details_next.html

```diff
@@ -2,46 +2,49 @@
 {% load lametro_extras %}
 
 <div>
-    <div>
-        <div class="row" style="margin-top: 1em;">
-            <div class="col-xs-6">
-                <img src='/static/images/Gateway03RT.jpg' class='img-responsive img-rounded' title='Los Angeles Gateway Plaza' alt="Los Angeles Gateway Plaza" />
-            </div>
-            <div class="col-xs-6">
-              <div class="row">
-                <div class="col-xs-12">
-                  <!-- Meeting name -->
-                  <h4>
-                    {{  meeting.link_html | safe }}<br/>
-                    <span class="small text-muted">{{ meeting.description }}</span>
-                  </h4>
+  <div>
+    <div class="row" style="margin-top: 1em;">
+      <div class="col-xs-6">
+        <img src='/static/images/Gateway03RT.jpg' class='img-responsive img-rounded' title='Los Angeles Gateway Plaza' alt="Los Angeles Gateway Plaza" />
+      </div>
+      <div class="col-xs-6">
+        <div class="row">
+          <div class="col-xs-12">
+            <!-- Meeting name -->
+            <h4>
+              {{  meeting.link_html | safe }}<br/>
+              <span class="small text-muted">{{ meeting.description }}</span>
+            </h4>
 
-                  <!-- Meeting info -->
-                  <p class="small text-muted">
-                      <i class="fa fa-fw fa-calendar-o"></i> {{ meeting.start_time | date:"D n/d/Y"}}<br/>
-                      <i class="fa fa-fw fa-clock-o"></i> {{ meeting.start_time | date:"g:i a"}}<br/>
-                      <i class="fa fa-fw fa-map-marker"></i> {{ meeting.location_name}}<br />
-                  </p>
-                  {% if meeting.documents.all %}
-                    <div class="col-xs-7">
-                      <a class="btn btn-teal" id="pdf-download-link" target='_blank' href='{{meeting.documents.all|find_agenda_url}}'><i class='fa fa-fw fa-download'></i> Get Agenda PDF</a>
-                    </div>
-                  {% endif %}
-                </div>
+            <!-- Meeting info -->
+
+            <p class="small text-muted">
+              <i class="fa fa-fw fa-calendar-o"></i> {{ meeting.start_time | date:"D n/d/Y"}}<br/>
+              <i class="fa fa-fw fa-clock-o"></i> {{ meeting.start_time | date:"g:i a"}}<br/>
+              <i class="fa fa-fw fa-map-marker"></i> {{ meeting.location_name}}<br />
+            </p>
+            {% if meeting.documents.all %}
+            <div class="row">
+              <div class="col-xs-7">
+                <a class="btn btn-teal" id="pdf-download-link" target='_blank' href='{{meeting.documents.all|find_agenda_url}}'><i class='fa fa-fw fa-download'></i> Get Agenda PDF</a>
               </div>
+              {% endif %}
```

Comment:
oops! if you open the row div inside the if statement, you should close it in there, too. otherwise you prematurely close the parent div if the if condition isn't satisfied. 
<img width="1295" alt="Screen Shot 2019-05-29 at 5 23 45 PM" src="https://user-images.githubusercontent.com/12176173/58595556-ae8f9100-8236-11e9-870b-0c2d2e45d755.png">